src/components/Contact.js

```diff
@@ -20,15 +122,15 @@ export default class Contact extends Component {
         >
           Contact Us
         </Header>
-
         <Form unstackable widths="equal">
           <Form.Group>
             <Form.Field>
               <Form.Input
                 placeholder="First Name"
                 label="First Name:"
                 required
-                value={this.state.fname}
+                value={this.state.value}
+                onChange={this.handleChangefName}
```

Comment:
add name='fName' to Form.Input, just above placeholder (firstName : if you follow the above comment)
use:  onChange={this.handleChange} instead